### PR TITLE
Add food link to attendee portal

### PR DESCRIPTION
**Pull request by alcarney**

### CODE REVIEW

File: src/attendee-portal.md [hunk 1]
```diff
@@ -34,12 +34,21 @@ title: Attendee Portal
 
 <div class="row">
 
+  <div class="box box_yellow">
+    <h3>What is for lunch?</h3>
+    <p>
+      The 
+      <a href="/food/">menu page</a>
+      has details on the available food and relevant dietry information.
```
Comment (geokala) [inline src/attendee-portal.md:42]: *dietary

**Comment by geokala** [comment]
Aside from that typo, this should be good to merge.

File: src/attendee-portal.md (hunk 1 above)
Comment (geokala) [inline src/attendee-portal.md:42]:
```suggestion
      has details on the available food and relevant dietary information.
```

**Comment by alcarney** [comment]
Thanks for getting this across the line!

**Pull request merged**